tools/run_tests/sanity/check_bazel_workspace.py

```diff
@@ -116,6 +118,9 @@ def git_repository(self, **args):
             return
         self.names_and_urls[args['name']] = args['remote']
 
+    def new_git_repository(self, **args):
```

Comment:
once you use http_archive in bazel_deps, you can revert the new_git_repository related changes.

Reply:
done